src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -476,12 +476,10 @@
     private Pattern publicMemberPattern = Pattern.compile("^serialVersionUID$");
 
     /** Set of ignore annotations short names. */
-    private Set<String> ignoreAnnotationShortNames =
-            getClassShortNames(DEFAULT_IGNORE_ANNOTATIONS);
+    private Set<String> ignoreAnnotationShortNames;
 
     /** Set of immutable classes short names. */
-    private Set<String> immutableClassShortNames =
-        getClassShortNames(DEFAULT_IMMUTABLE_TYPES);
+    private Set<String> immutableClassShortNames;
```

Comment:
These 2 are similar to the previous. They are assigned default values in the `beginTree`.

`immutableClassShortNames` is given `immutableClassCanonicalNames`, and `immutableClassCanonicalNames`'s default value is `DEFAULT_IMMUTABLE_TYPES`.
`ignoreAnnotationShortNames` is given `ignoreAnnotationCanonicalNames`, and `ignoreAnnotationCanonicalNames`'s default value is `DEFAULT_IGNORE_ANNOTATIONS`.